packages/app/src/runner/event-manager.ts

```diff
@@ -54,6 +55,8 @@ export class EventManager {
   studioRecorder: any
   selectorPlaygroundModel: any
   cypressInCypressMochaEvents: CypressInCypressMochaEvent[] = []
+  // Used for testing the experimentalSingleTabRunMode experiment. Ensures AUT is correctly destroyed between specs.
+  autDestroyedCount = 0
```

Comment:
Any way we can setup this up in the e2e/ct support files so we don't have this hanging around in prod code?

Reply:
Sure thing, nice idea. Done: https://github.com/cypress-io/cypress/pull/23104/commits/3cb9ab34205e892cca64c453ff92a1cdf15d208d